docs/dev/api/api_samples.md

```diff
@@ -79,13 +79,13 @@
 	"_embedded": {
 		"csar": [
 			{
-				"name": "k8s-cluster",
+				"name": kubernetes,
 				"_links": {
 					"self": {
-						"href": "http://localhost:8080/csars/k8s-cluster"
+						"href": kubernetes
```

Comment:
Hmm this is not the proper value.

Reply:
Ou something with refactoring went wrong.